tests/testing_helpers.py

```diff
@@ -405,7 +407,9 @@ def build_real_server(global_conf=None, main_conf=None):
     )
 
 
-def build_mocked_server(name=None, config=None, global_conf=None, main_conf=None):
+def build_mocked_server(
+    name=None, config=None, global_conf=None, main_conf=None, pg_version=170000
```

Comment:
```suggestion
    name=None, config=None, global_conf=None, main_conf=None, pg_version=None
```

I'd say to make it default to `None`, then adjust the unit tests so they set the version they expect for a given unit test.

This way we have a more explicit setting in the unit test which requires this information, if you get what I mean?

Reply:
@barthisrael I had a quick call with @andremagui yesterday, and I suggested setting the value to 17000 as seemed to me at a glance, the less impactful way to implement this change.

Looking deeper into the code today, the above suggestion seems sensible and more correct. I recommend it too.

Reply:
Thanks for the heads up, got it!
I mentioned about using `None` because it would remain backward compatible with the existing tests, and would "force" us to choose it appropriately in the unit tests where that thing is a constraint.